### ISSUE TITLE
When 'show seconds' is enabled, seconds count downwards from 59 to 00 instead of expected behavior of counting upwards.

### ISSUE THREAD

**Issue by TheChronoMaster** [unsoluble/smalltime#97]
Using PF2e, Simple Calendar, SmallTime, no other time mods. Calendar provider is set to 'Simple Calendar'. Whether clock is set to 24h or 12h, as long as 'show seconds' is enabled, and time is progressing in real time, the seconds count in reverse. Minutes and hours advance as normal despite this behavior.

**Comment by unsoluble** [comment]
Ah. Bloody weird PF2E clock stuff strikes again — this'd be because you're set to a data that's prior to the world-start epoch for your world (so it's acting like BCE and counting down toward zero). I'll see what I can do about accounting for this, but will probably be a few days before I can get to it.

**Comment by TheChronoMaster** [comment]
Interesting, because my date is set well after the world-start epoch (by several years), and it does SEEM to properly count minutes and hours upwards.

**Comment by unsoluble** [comment]
So the minute-off thing is an interaction with Simple Calendar that I haven't sorted out yet.

The hours/minutes count up correctly because they're using a different mechanism from the seconds (for reasons).

Are you sure your date is after _your_ world start epoch? (not the PF2 canonical start date, but the date you get in that specific world when resetting the clock)

**Comment by TheChronoMaster** [comment]
Okay, I think you are correct - the world's start date is always set to the exact equivalent time as when the world is first created IRL (Meaning current date, +2700 years, changing the names of months and days - in my case, this is Rova 11, 4721 AR, as opposed to the date I had set for the game, Lamashan in 4719). Even if 'current year' is set to something else in Simple Calendar, this appears to be the case.

From a practical perspective, I don't think this really affects much for my purposes, but it does seem very strange that it is calculated based on when I created the game world, rather than what I have the game world's time and year 0 set to.
